src/pages/OperationsPage.js

```diff
@@ -47,10 +48,12 @@ class OperationsPage extends Component {
     }
 
     componentWillMount(){
-        this.props.getConfirmationsOnce();
-        this.props.getListCBE();
-        this.props.getPendingsOnce();
-        this.props.getLOCsOnce();
+        this.props.getConfirmationsOnce();   //  todo
+        this.props.getListCBE();   //  todo
+        this.props.getPendingsOnce();   //  todo
+        if (this.props.isLOCsNeedReload) {
```

Comment:
This decision is too hasty, it is better to leave it for later, and not apply to this PR